platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/command/ProfileInstallCommand.java

```diff
@@ -0,0 +1,231 @@
+                    return true;
+                });
+                try {
+                    if (featuresService.isInstalled(featuresService.getFeature(feature))) {
+                        printItemStatusSuccess("Installed: ", feature);
+                    } else {
+                        printItemStatusFailure("Install Failed: ", feature);
+                    }
+                } catch (Exception e) {
+                    printError("An error occurred when checking if feature installed successfully");
+                    LOGGER.error("Error occurred when checking if feature was installed successfully: %s", e);
+                }
+            });
+        }
+    }
+
+    private void uninstallFeatures(List<String> uninstallFeatures) {
+        if (uninstallFeatures != null) {
+            printSectionHeading("Uninstalling Features");
+            uninstallFeatures.forEach(feature -> {
+                printItemStatusPending("Uninstalling: ", feature);
+                executeAsSystem(() -> {
+                    Feature featureObject = featuresService.getFeature(feature);
+                    featuresService.uninstallFeature(featureObject.getName(),
+                            featureObject.getVersion(),
+                            EnumSet.of(FeaturesService.Option.NoAutoRefreshBundles));
+                    return true;
+                });
+                try {
+                    if (!featuresService.isInstalled(featuresService.getFeature(feature))) {
+                        printItemStatusSuccess("Uninstalled: ", feature);
+                    } else {
+                        printItemStatusFailure("Uninstall Failed: ", feature);
+                    }
+                } catch (Exception e) {
+                    printError("An error occurred when checking if feature uninstalled successfully");
+                    LOGGER.error("Error occurred when checking if feature was uninstalled successfully: %s", e);
+                }
+            });
+        }
+    }
+
+    private void stopBundles(List<String> stopBundleNames) {
+        if (stopBundleNames != null) {
+            BundleService bundleService = (BundleService) getService(BundleService.class);
+            if (bundleService != null) {
+                printSectionHeading("Stopping Bundles");
+                stopBundleNames.forEach(bundle -> {
+                    printItemStatusPending("Stopping: ", bundle);
+                    executeAsSystem(() -> {
+                        bundleService.getBundle(bundle).stop();
+                        return true;
+                    });
+                    if (bundleService.getBundle(bundle).getState() == Bundle.RESOLVED) {
+                        printItemStatusSuccess("Stopped: ", bundle);
+                    } else {
+                        printItemStatusFailure("Stop Failed: ", bundle);
+                    }
+                });
+            }
+        }
+    }
+
+    private Object getService(Class clazz) {
```

Comment:
This method has now been removed since all services are now found via `@Reference` instead